src/webview/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -163,6 +168,16 @@ protected void onStart() {
 		super.onStart();
 	}
 
+	@Override
+	protected void onStop() {
+		super.onStop();
+		try {
+			settings.setLastUrl(container.getUrl());
```

Comment:
Do you think we need to do any protection to make sure it's still going to the right domain? For example, if someone clicks a link that takes them to google.com, then closes and opens the app, there will be no obvious way to go back to the CHT. It might be worth checking that the domain of the start URL matches the domain of the last URL...

Reply:
Good point ! Do you think that maybe we should also record the timestamp when was recorded the last URL? so if the recording was time ago we just ignore it, because it may be confusing to enter the app a week later and the app going straight to the last form visited, right? maybe only reload the last URL if the form was recorded the last 24 hours or less.

Reply:
Maybe... I'm not too worried about it because as long as you're still in the CHT then it's easy to get to the page you want. But if you think that'd be an improvement, go for it!